app/models/manageiq/providers/redhat/infra_manager/vm_import.rb

```diff
@@ -60,9 +89,14 @@ def perform_vmware_to_ovirt_import(params)
           :sparse         => params[:sparse],
         )
       )
+      ems_ref_for_vm_id(import.vm.id)
```

Comment:
I don't believe so, If the API returns an 20x OK status it should contain the basic vm ID data